cpp/src/quantiles/group_quantiles.cu

```diff
@@ -79,6 +80,40 @@ struct quantiles_functor {
 
 } // namespace anonymous
 
+namespace detail {
+
+void group_quantiles(gdf_column const& values,
+                     rmm::device_vector<gdf_size_type> const& group_offsets,
+                     rmm::device_vector<gdf_size_type> const& group_sizes,
+                     gdf_column * result,
+                     std::vector<double> const& quantiles,
+                     cudf::interpolation interpolation,
+                     cudaStream_t stream)
+{
+  rmm::device_vector<double> dv_quantiles(quantiles);
+
+  type_dispatcher(values.dtype, quantiles_functor{},
+                  values, group_offsets, group_sizes, result,
+                  dv_quantiles, interpolation, stream);
+}
+
+void group_medians(gdf_column const& values,
```

Comment:
Should it be called `group_median` since there's only one median per group? The python API is `groupby.median()`, not medians. (`group_quantiles` on the other hand returns multiple quantiles per group, right?) 

```suggestion
void group_median(gdf_column const& values,
```

Reply:
@devavret can you help us to answer this part?

Reply:
it returns multiple values that can be best described as `median_of_each_group()`. The other candidate was `groups_median`, which makes it sound like we're finding a group that's in the middle of all the groups.